test/functional/no-filter.test.ts

```diff
@@ -70,10 +69,8 @@ describe('no filters broker', () => {
     );
 
     expect(response.status).toEqual(401);
-    expect(response.data).not.toBe({
-      message: 'blocked',
-      reason: 'Request does not match any accept rule, blocking HTTP request',
-      url: '/echo-server',
+    expect(response.data).not.toStrictEqual({
```

Comment:
Not sure I like this assertion, kinda prefer the one before in principle

Reply:
Agree, fixed with 5c4112a.